### DISCUSSION TITLE
the tagged files reappear in ”Batch Operations"

### DISCUSSION THREAD

**Discussion by yushizui** [Difegue/LANraragi#1004] [Q&A] [answered]
**LRR Version and OS**
0.9.1 docker
**Bug Details**

After the update, the tagged files reappear in ”Batch Operations“
They are all comics that have been deleted from ehentai. I have manually added tags, "source: no source "and other tag

**Comment by Difegue** [comment]
What do you mean by "reappear"? Please provide more details.

**Comment by yushizui** [comment]
![image](https://github.com/Difegue/LANraragi/assets/130179532/a016e4ca-9ebb-4c07-9d4c-13fb218e9d23)
After adding the source: tag before updating, it will not appear in "Batch Operation"

**Comment by Difegue** [comment, marked as answer]
Thanks for clarifying -- Due to #992, `source:` is no longer considered a namespace that excludes archives from the untagged list.  
You can likely use tag rules to replace your `source: no source` tags with anything else without the namespace to fix this.
